makejdk.sh

```diff
@@ -75,6 +75,9 @@ while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
 
     "--jtreg" | "-j" )
     JTREG=true; shift;;
+    
+    "--jtreg_subsets" )
+    JTREG=true; JTREG_TEST_SUBSETS="$1"; shift;;
```

Comment:
After playing with the shell script, I've determined that shifting after assigning $1 to it has the nasty side-effect of swallowing the next variable passed in on the command line.

so "./makejdk.sh --jtreg --potato" assigns "--potato" to JTREG_TEST_SUBSETS, and removes --potato from the list of arguments to be processed.

If we don't want to add another "if", what we could do is require a value for jtreg.

'./makejdk.sh --jtreg "" --potato' works, for example.

Will make that change. Let me know what you think when you get back.